### PR TITLE
Add angular config for template updating

### PR DESCRIPTION
**Pull request by egaluza**

### CODE REVIEW

File: templates-generator/angular-config.js [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+                    before: /@ViewChild\(DxTreeViewComponent.*?\)/,
+                    after: '@ViewChild(DxTreeViewComponent<% if(requireStaticFlag) { %>, { static: true }<% } %>)'
+                }
+            ]
+        },
+        {
+            glob: 'src/app/app-routing.module.ts',
+            definitions: [
+                {
+                    before: /import { HomeComponent } from '.\/pages\/home\/home.component';[\r\n]/,
```
Comment (nightskylark) [inline templates-generator/angular-config.js]: Why you are using regular expression here?

Comment (egaluza) [reply templates-generator/angular-config.js]: Regular Expression will work with different line endings

Comment (egaluza) [reply templates-generator/angular-config.js]: I replaced expression

**Comment by egaluza** [comment]
[Add angular config for update-templates script](https://trello.com/c/41CuXHhV/2154-add-angular-config-for-update-templates-script)

**Pull request closed without merging**